baby-bear/benches/bench_field.rs

```diff
@@ -33,5 +36,20 @@ fn bench_field(c: &mut Criterion) {
     });
 }
 
-criterion_group!(baby_bear_arithmetic, bench_field);
+fn bench_packedfield(c: &mut Criterion) {
+    let name = type_name::<<F as Field>::Packing>().to_string();
+    // Note that each round of throughput has 10 operations
+    // So we should have 10 * more repetitions for latency tests.
+    const REPS: usize = 100;
+    const L_REPS: usize = 10 * REPS;
+
+    // benchmark_add_latency::<<F as Field>::Packing, L_REPS>(c, &name);
+    // benchmark_add_throughput::<<F as Field>::Packing, REPS>(c, &name);
+    // benchmark_sub_latency::<<F as Field>::Packing, L_REPS>(c, &name);
+    // benchmark_sub_throughput::<<F as Field>::Packing, REPS>(c, &name);
```

Comment:
I think these should either be uncommented, right?

Reply:
Yep. Whoops, I'll fix that before the merge.